requirements/requirements-mxnet.txt

```diff
@@ -1 +1 @@
-mxnet~=1.7
+mxnet==1.7.*
```

Comment:
I think we just changed the requirements to mxnet in another PR and this should be aligned.

Reply:
This looks a bit strange, since this should be the same as what is on master, and thus doesn't show in the diff.

Reply:
Yes, I'm not sure what that was about. But with the new commits this longer appears as a change.